src/components/VideoPlayerContexts/PlaybackContext.js

```diff
@@ -67,6 +67,9 @@ function PlaybackContextProvider({children}) {
 
     const checkVideoPlaying = useCallback(
         (statusCallback) => {
+            if (!(currentVideoPlayerRef && currentVideoPlayerRef.current && currentVideoPlayerRef.current.getStatusAsync)) {
```

Comment:
We need to check if the function isn't null, it it is we can safely keep status default value (which is `false`).